### DISCUSSION TITLE
Welcome to curv Discussions!

### DISCUSSION THREAD

**Discussion by doug-moen** [curv3d/curv#106] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wimdows** [comment]
Hello Doug.
I just discovered Curv. And I must say, I find it intriguing.
Curv seems to offer endless posibilities for visualizing mathematics. And I am looking forward to exploring them.
Hopefully more people will discover your programming language, and will join these conversations.

Thanks a bundle for all the work you put into Curv!

Best regards,

wim
